src/index.tsx

```diff
@@ -36,7 +36,7 @@ export class ApptentiveConfiguration {
     this.apptentiveSignature = signature;
     this.logLevel = "info";
     this.distributionName = "React Native";
-    this.distributionVersion = require("../package").version;
```

Comment:
Do we know if this was causing issues? Would prefer not to introduce another copy of the version (beyond the one in package.json), but if it's unreliable maybe we have to.

Reply:
I don't think it would be the cause of the issue. I had series of issues with RattlsesnakeFax not recognizing a list of files and assets that are present in the node-modules. Had a tough time trying to fix them all by taking a different route. Today it was all magically solved. I was doing the same thing rm -rf node-modules/ , clear cache etc., don't know how it got solved today. Just forgot to undo this change in the plugin.